mscore/musescorefonts-Mac.qrc

```diff
@@ -1,10 +1,12 @@
 <RCC>
     <qresource prefix="/">
         <file alias="fonts/leland/Leland.otf">../fonts/leland/Leland.otf</file>
+        <file alias="fonts/bravura/Bravura.otf">../fonts/bravura/Bravura.otf</file>
         <file alias="fonts/mscore/mscore.ttf">../fonts/mscore/mscore.ttf</file>
         <file alias="fonts/gootville/Gootville.otf">../fonts/gootville/Gootville.otf</file>
-        <file alias="fonts/bravura/Bravura.otf">../fonts/bravura/Bravura.otf</file>
         <file alias="fonts/musejazz/MuseJazz.otf">../fonts/musejazz/MuseJazz.otf</file>
         <file alias="fonts/petaluma/Petaluma.otf">../fonts/petaluma/Petaluma.otf</file>
+        <file alias="fonts/finalemaestro/FinaleMaestro.otf">../fonts/finalemaestro/FinaleMaestro.otf</file>
+        <file alias="fonts/finalebroadway/FinaleBroadway.otf">../fonts/finalbroadway/FinaleBroadway.otf</file>
```

Comment:
So this is the problem:
```suggestion
        <file alias="fonts/finalebroadway/FinaleBroadway.otf">../fonts/finalebroadway/FinaleBroadway.otf</file>
```

Reply:
Uhhh, what a stupid typo. Thanks for pointing it out ;-)